extensions/ql-vscode/src/extension.ts

```diff
@@ -156,6 +158,11 @@ export interface CodeQLExtensionInterface {
  * @returns CodeQLExtensionInterface
  */
 export async function activate(ctx: ExtensionContext): Promise<CodeQLExtensionInterface | Record<string, never>> {
+
+  const codelensProvider = new QuickEvalCodeLensProvider();
+
+  languages.registerCodeLensProvider({ scheme: 'file', language: 'ql' }, codelensProvider);
```

Comment:
Please move this further down in the function: I don't think code lens creation should happen before we call `languageSupport.install()`, and certainly not before we've set up the extension and logging.

Reply:
Moved it to be after `languageSupport.install()`, but if you think it should go further down in the function I can move it again.